supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/OccasionalStatusPollingActor.scala

```diff
@@ -0,0 +1,89 @@
+package cromwell.backend.impl.aws
+
+import akka.actor.{Actor, ActorLogging, Props}
+import cromwell.backend.impl.aws.OccasionalStatusPollingActor._
+import cromwell.backend.impl.aws.RunStatus.{Initializing, Running}
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.batch.BatchClient
+import software.amazon.awssdk.services.batch.model.ListJobsRequest
+
+import scala.collection.JavaConverters._
+import scala.concurrent.duration._
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success}
+
+class OccasionalStatusPollingActor(configRegion: Option[Region]) extends Actor with ActorLogging {
+
+  implicit val ec: ExecutionContext = context.dispatcher
+
+  val PollingInterval = 10.seconds
+
+  lazy val client = {
+    val builder = BatchClient.builder()
+    configRegion.foreach(builder.region)
+    builder.build
+  }
+
+  // Maps job ID to status
+  var statuses: Map[String, RunStatus] = Map.empty
+
+  override def receive = {
+    case WhatsMyStatus(jobId) =>
+      sender ! ThisWasYourStatus(statuses.get(jobId))
+
+    case UpdateStatuses =>
+      Future {
+        updateStatuses()
+      } onComplete {
+        case Success(_) =>
+          scheduleStatusUpdate(PollingInterval)
+        case Failure(error) =>
+          log.error(error, "Failed to update statuses. Will try again in 2 seconds")
+          scheduleStatusUpdate(2.seconds)
+      }
+    case NotifyOfStatus(jobId, status) => statuses += jobId -> status
+  }
+
+  private def updateStatuses() = {
+
+    def updateForStatus(awsStatusNames: Seq[String], runStatus: RunStatus) = {
+
+      val valuesInStatus = awsStatusNames flatMap { awsStatusName =>
+        val request = ListJobsRequest.builder()
+          .jobStatus(awsStatusName)
+          .jobQueue("GenomicsDefaultQueue-9d1df58e6cbf7dc")
```

Comment:
everybody gets to use our queue?

Reply:
Good spot! This probably explains why the Hutch folks saw no improvements even when they tried out this branch